src/gap.c

```diff
@@ -1597,6 +1597,9 @@ static Int InitKernel (
     /* list of exit functions                                              */
     InitGlobalBag( &WindowCmdString, "src/gap.c:WindowCmdString" );
 
+    InitGlobalBag( &STATE(ShellContext), "STATE(ShellContext)" );
+    InitGlobalBag( &STATE(BaseShellContext), "STATE(BaseShellContext)" );
```

Comment:
I don't think these, nor the one for `ErrorLVars`, are strictly necessary, but they won't hurt, and make it clear that everything is in order w/o forcing the reader to deeply analyze the code, so I think it's a good idea to add the,